### DISCUSSION TITLE
Multiselect is not working for Bootstrap V5

### DISCUSSION THREAD

**Discussion by harishkoninti** [davidstutz/bootstrap-multiselect#1202] [General]
I am using Bootstrap 5 and I want to use Bootstrap-multi select but it is not working. will it work with Bootstrap 5, is that Possible?

**Comment by tiesont** [comment]
As-is, no. Bootstrap 5 changes the triggering attributes (`data-bs-whatever` rather than `data-whatever`), so there's some work to do to support Bootstrap 5. I think those updates are in the works, but there's no ETA.

**Reply by harishkoninti** [reply]
@tiesont  Thanks for the information.

**Comment by halimus** [comment]
@harishkoninti, what I did on my side: in bootstrap-multiselect.js

line 407: change **buttonClass: 'custom-select',** into: **buttonClass: 'form-select',**
line 450: change **data-toggle="dropdown"** into: **data-bs-toggle="dropdown"**

and it seems working fine with bootstrap 5.0.2

**Reply by lojacurio** [reply]
Fiz as alterações conforme solicitado e funcionou bem usando o bootstrap 5.02.

**Reply by RasmusDamgaard** [reply]
This worked for me! Thank you!

**Reply by Sed2295** [reply]
Thank´s :)

**Comment by FVZGmbH** [comment]
line 521 + 522: change 'left' into: 'start'
line 527 + 528: change 'right' into: 'end'

**Comment by AnadarPro** [comment]
Unfortunately in 5.1.3 these didn't help.

**Comment by nonprofittechy** [comment]
This is a working compiled version. Need to do a proper fork and probably will end up maintaining our own copy: https://github.com/SuffolkLITLab/docassemble-ALToolbox/blob/main/docassemble/ALToolbox/data/static/bootstrap-multiselect.js
